Sources/ClientRuntime/EventStream/DefaultMessageEncoderStream.swift

```diff
@@ -9,7 +9,7 @@ import struct Foundation.Data
 
 extension EventStream {
     /// Stream adapter that encodes input into `Data` objects.
-    public class DefaultMessageEncoderStream<Event: MessageMarshallable>: MessageEncoderStream, Stream {
+    public class DefaultMessageEncoderStream<Event>: MessageEncoderStream, Stream {
```

Comment:
The `MessageMarshallable` protocol is removed in favor of marshal/unmarshal closures.